src/css/visualization.css

```diff
@@ -196,6 +196,11 @@
   background-color: rgba(193, 78, 248, 1);
 }
 
+.nominal-bar {
+  /* Placeholder */
+  background-color: rgba(238, 238, 238, 1);
```

Comment:
Let's wait to merge this until Rong gives us the color she wants.

Reply:
Removed from this PR - will do this in another PR.